airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionForm.tsx

```diff
@@ -376,7 +378,11 @@ export const ConnectionForm: React.FC<ConnectionFormProps> = ({
                 isSubmitting={isSubmitting}
                 isValid={isValid && !editingTransformation}
                 errorMessage={
-                  errorMessage || !isValid ? formatMessage({ id: "connectionForm.validation.error" }) : null
+                  errorMessage
+                    ? errorMessage
+                    : !isValid
+                    ? formatMessage({ id: "connectionForm.validation.error" })
+                    : null
```

Comment:
I feel like, since this is repeated, you can just do an:
`const displayedErrorMessage = errorMessage ?? !isValid ? formatMessage({ id: "connectionForm.validation.error" }) : null;`
and use that in both places

Reply:
+1